bicep/main.bicep

```diff
@@ -198,6 +198,15 @@ module dnsZone './dnsZone.bicep' = if (!empty(dnsZoneId)) {
 @description('Installs the AKS KV CSI provider')
 param azureKeyvaultSecretsProvider bool = false
 
+@description('Rotation poll interval for the AKS KV CSI provider')
+@allowed([
+  '2m'
```

Comment:
Is 2 minutes the **minimum** rotation-poll interval?
Is there a max?

Reply:
I like the fact that the UI is helpful with its options for rotation intervals, but think we need to yank it out of the `@allowed` bicep decorator to allow for customers to specify `4m`, `60m` or even `30s` which seems to work.

@liammoat - Are you ok to remove the allowed decorator from the bicep parameter?

Reply:
@Gordonby - yep, I'll update today.